app/api/shared.js

```diff
@@ -0,0 +1,19 @@
+const wreck = require('@hapi/wreck')
```

Comment:
Makes very good sense to extract these methods out to a shared files. Good thinking